scripts/makeOVPN.sh

```diff
@@ -269,15 +269,16 @@ echo "tls-auth Private Key found: $TA"
 } > "${NAME}${FILEEXT}"
 
 # Copy the .ovpn profile to the home directory for convenient remote access
-cp "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT" "/home/$INSTALL_USER/ovpns/$NAME$FILEEXT"
-chown "$INSTALL_USER" "/home/$INSTALL_USER/ovpns/$NAME$FILEEXT"
+INSTALL_HOME=$(cat /etc/passwd | grep "$INSTALL_USER" | cut -d: -f6)
+cp "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT" "$INSTALL_HOME/ovpns/$NAME$FILEEXT"
+chown "$INSTALL_HOME" "$INSTALL_HOME/ovpns/$NAME$FILEEXT"
 chmod o-r "/etc/openvpn/easy-rsa/pki/$NAME$FILEEXT"
-chmod o-r "/home/$INSTALL_USER/ovpns/$NAME$FILEEXT"
+chmod o-r "$INSTALL_HOME/ovpns/$NAME$FILEEXT"
```

Comment:
rather use numerical permission, these are more concrete, easier to read and understand, can you change this to chmod 640 ?

Reply:
Tis done.... Thanks!